tests/transports-governance/test_governance_hierarchical_models.py

```
@@ -0,0 +1,908 @@
+            budget_data = test_data_factory.budget_data(
+                max_limit=case["max_limit"], reset_duration=case["reset_duration"]
+            )
+            response = governance_client.create_budget(**budget_data)
+
+            assert_valid_response(response, 201)
+            data = response.json()
+            assert data["max_limit"] == case["max_limit"]
+
+            track_created_entity("budgets", data["id"])
+
+    def test_create_budget_invalid_values(self, governance_client, test_data_factory):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_create_budget_invalid_values`

Add return type annotation: `None`

(ANN201)